samples/data_quality/README.md

```diff
@@ -0,0 +1,50 @@
+# Bring Your Own Container Sample
+
+This sample demonstrates how to start data-quality jobs using your own data-quality script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](TODO).
+
+In order to follow this script, you must first create a data-quality script packaged in a Dockerfile that is [compatible with Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/amazon-sagemaker-containers.html).
```

Comment:
For data quality, model quality and model explainability, the customers will also need to run baselining jobs using the Python SageMaker SDK before they can use it with ACK. Only that SDK offers the tools to create a baseline metrics package that is required in the spec. (See `dataQualityBaselineConfig` fields)